tests/components/controlBar-test.js

```diff
@@ -129,136 +129,155 @@ describe('ControlBar', function() {
     expect(fullscreenToggled).toBe(true);
   });
 
-  it('render one stereo button if content vr', function() {
-    var mockController = {
-      state: {
-        isMobile: true,
-        volumeState: {
-          volume: 1
+  describe('Vr on phones', function(){
+    beforeEach(function() {
+      window.navigator.userAgent = 'phone';
+    });
+    afterEach(function() {
+      window.navigator.userAgent = 'desktop';
+    });
+
+    it('render one stereo button if content vr', function() {
+      var mockController = {
```

Comment:
Can we use baseMockController and baseMockProps and only set the properties that are needed for the test? This will help eliminate boiler plate code and keep tests short. I recommend the same for the tests below.

Reply:
\+